code/modules/antagonists/changeling/powers/mimic_voice.dm

```diff
@@ -4,7 +4,7 @@
 	button_icon_state = "mimic_voice"
 	helptext = "Will turn your voice into the name that you enter. We must constantly expend chemicals to maintain our form like this."
 	chemical_cost = 0//constant chemical drain hardcoded
-	dna_cost = 1
+	dna_cost = CHANGELING_POWER_INNATE
```

Comment:
This is a weird thing to bundle with your other changes, but you should just make it free instead of innate.
Changelings already start with a lot of buttons, don't shove another one on there which they may not actually ever use.

Reply:
sure thing